### PR TITLE
Fix copy-paste error in updateVersion

### PR DESCRIPTION
**Pull request by hpinkos**
This was using the incorrect variable

### CODE REVIEW

**Comment by mramato** [comment]
We use the `2.0` branch of `gltf-pipeline` in Cesium.  Is that were this is from?

**Comment by hpinkos** [comment]
Yes, sorry, I should have specified that in my PR comment.  This is from the current 2.0 branch

**Comment by lilleyse** [comment]
This code actually belongs here but needs to be ported back into gltf-pipeline. There may be one or two other cases like this that started as Cesium PRs and never made their way back. I'll look into it.

**Comment by hpinkos** [comment]
Oh, nevermind, it looks like @lilleyse  updated is file in Cesium but not in `gltf-pipeline` https://github.com/AnalyticalGraphicsInc/cesium/commit/e3357232428ceb0db2db9f7c7a86b070ab0e6681#diff-ba60174ac65ae60b89c9b21903710cae

@lilleyse what is the correct thing to do here?

File: Source/ThirdParty/GltfPipeline/updateVersion.js [hunk 1]
```diff
@@ -508,26 +508,6 @@ define([
...
-                        var compressedImage = compressedImages[type];
-                        var compressedExtensions = compressedImage.extensions;
-                        if (defined(compressedExtensions)) {
-                            var compressedBinaryGltf = compressedExtensions.KHR_binary_glTF;
-                            if (defined(compressedBinaryGltf)) {
-                                compressedImage.bufferView = globalMapping.bufferViews[compressedBinaryGltf.bufferView];
-                                compressedImage.mimeType = compressedBinaryGltf.mimeType;
-                                delete compressedExtensions.KHR_binary_glTF;
-                            }
-                            if (Object.keys(extensions).length === 0) {
```
Comment (hpinkos) [inline Source/ThirdParty/GltfPipeline/updateVersion.js]: @lilleyse also, this looks like a copy-paste error.  should this be `compressedExtensions`?

Comment (lilleyse) [reply Source/ThirdParty/GltfPipeline/updateVersion.js]: Oh, yeah I think so.

**Comment by hpinkos** [comment]
@lilleyse this is ready

**Pull request merged**